io.github.sil_car.SqueezeVid.yaml

```diff
@@ -0,0 +1,133 @@
+
+  - name: libvpx
+    # https://github.com/webmproject/libvpx
+    sources:
+      - type: git
+        url: https://chromium.googlesource.com/webm/libvpx.git
+        tag: v1.11.0
+        commit: 626ff35955c2c35b806b3e0ecf551a1a8611cdbf
+    buildsystem: autotools
+    builddir: true
+    config-opts:
+      # https://github.com/webmproject/libvpx/blob/main/configure
+      - --disable-docs
+      - --disable-examples
+      - --disable-static
+      - --disable-unit-tests
+      - --enable-shared
+      - --enable-multithread
+
+  - name: libx264
+    sources:
+      - type: git
+        url: https://code.videolan.org/videolan/x264.git
+        branch: stable
+        commit: baee400fa9ced6f5481a728138fed6e867b0ff7f
+    buildsystem: autotools
+    config-opts:
+      # https://code.videolan.org/videolan/x264/-/blob/master/configure
+      - --disable-cli
+      - --disable-static
+      - --enable-shared
+
+  - name: ffmpeg
+    # Other prereqs?:
+    #   - libsdl1.2: https://github.com/flathub/shared-modules/tree/master/SDL
+    sources:
+      - type: git
+        url: https://git.videolan.org/git/ffmpeg.git
+        tag: n6.0
+        commit: ea3d24bbe3c58b171e55fe2151fc7ffaca3ab3d2
+    buildsystem: autotools
+    config-opts:
+      # https://gist.github.com/omegdadi/6904512c0a948225c81114b1c5acb875
+      - --disable-debug
+      - --disable-doc # removes all docs --disable-{htmlpages,manpages,txtpages}
+      - --disable-ffplay
+      - --disable-static
+      - --enable-gpl
+      - --enable-libaom         # AV1 video codec (for decode support)
+      - --disable-encoder=libaom
+      - --enable-libass         # subtitles support
+      - --enable-libsvtav1      # SVT-AV1 video codec; requires 22.04+ packages libsvtav1{,-dev}; or build from src
+      - --enable-libvpx         # VP9 video codec
+      - --enable-libx264        # H.264 video codec
+      - --enable-opengl
+      - --enable-shared
+      - --enable-version3
+
+  - python3-requirements.yaml
+
+  - name: squeeze-vid
+    sources:
+      - type: git
+        url: https://github.com/sil-car/squeeze-vid.git
+        tag: v0.5.2
+        commit: bb833d3ea7013720eb2c1a4dba93e229969e228b
+    buildsystem: simple
+    build-commands:
+      # Ref: https://github.com/pypa/pip/issues/7730 (use --no-build-isolation to access host setuptools)
+      - pip3 install --no-build-isolation --prefix="${FLATPAK_DEST}" .
+      # Install packaging files.
+      - install -Dm644 pkg-data/io.github.sil_car.SqueezeVid.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo
+      - install -Dm644 pkg-data/video-editor_64.png ${FLATPAK_DEST}/share/icons/hicolor/64x64/apps/io.github.sil_car.SqueezeVid.png
+      - install -Dm644 pkg-data/video-editor_128.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/io.github.sil_car.SqueezeVid.png
+      - install -Dm644 pkg-data/video-editor_128.png ${FLATPAK_DEST}/share/app-info/icons/flatpak/128x128/io.github.sil_car.SqueezeVid.png
```

Comment:
```suggestion
```

NEVER put anything there.

Reply:
What am I doing wrong, then, since it's checking for an icon there? Sorry, I'm a first time flatpak creator...

Reply:
https://github.com/flathub/flathub/pull/4335#issuecomment-1647747001

Reply:
and https://github.com/flathub/flathub/pull/4335#issuecomment-1648044476